docs/docs/components/avatar.md

```diff
@@ -0,0 +1,77 @@
+<!-- This file is generated through a template. Do not edit directly. -->
+<!-- generated by ui/tools/generator_doc_components_page.js  -->
+<script setup>
+	import { data as components } from "../components.data.ts";
+	const component = components.find(c => c.name === "Avatar");
+</script>
+
+<h1>{{ component.name }}</h1>
```

Comment:
I should use the markdown inclusion to keep the generated part as light as possible

https://vitepress.dev/guide/markdown#markdown-file-inclusion